src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -476,10 +476,24 @@ public function __clone()
      */
     public function __toString()
     {
+        $protocol = '';
+        try {
+            $protocol = $this->server->get('SERVER_PROTOCOL');
+        } catch (\InvalidArgumentException $mallformedPath) {
```

Comment:
We merged a PR where all caught exceptions were renamed to $e. So should be this one.